### PR TITLE
Data/fetch user data

### PR DESCRIPTION
**Pull request by Melizzo**
# PR Description
Please include a summary of the change:
Fetched wcUserData/ingredientsData, recipesData

Commit Emojis:

:tada: Initial commit
:crystal_ball: JS modifications/updates.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
switched data to API data, and updated new user instantiation. 

Note any questions or struggles as well.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -1,35 +1,76 @@
...
+.then(data => data.ingredientsData)
+.catch(err => console.log(err.message))
+
+recipeData = fetch('https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/recipes/recipeData')
+.then(data => data.json())
+.then(data => data.recipeData)
+.catch(err => console.log(err.message))
+
+//PROMISE
+Promise.all([wcUsersData, ingredientsData, recipeData])
```
Comment (Billwilke42) [inline src/scripts.js:35]: Looks great, stoked everything's fetching!

File: src/scripts.js [hunk 2]
```diff
@@ -1,35 +1,76 @@
...
 cardArea.addEventListener('click', cardButtonConditionals);
 
-function onStartup() {
-  let userId = (Math.floor(Math.random() * 49) + 1)
-  let newUser = users.find(user => {
-    return user.id === Number(userId);
-  });
-  user = new User(userId, newUser.name, newUser.pantry)
-  pantry = new Pantry(newUser.pantry)
+function onStartup(wcUsersData) {
```
Comment (Billwilke42) [inline src/scripts.js:60]: Nice adjustments to the onStartup method!

**Comment by jeremiahblackol** [comment]
This looks great!

File: src/scripts.js (hunk 2 above)
Comment (jeremiahblackol) [reply src/scripts.js:60]: Dope, let's remember to check back in on their naming conventions, once we're up and running like we want to be.

**Pull request merged**